File: src/SpecializedClinicApplication/Model/ClientModel.cs
using SpecializedClinicApplication.Data.Models.Inventory;

namespace SpecializedClinicApplication.Model;
public class ClientModel
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Address { get; set; }
    public string Email { get; set; }
    public int PhoneNumber { get; set; }
}

File: src/SpecializedClinicApplication/ClientApp/src/models/client.model.ts
export class Client {

  constructor(id?: number, name?: string, address?: string, email?: string, phoneNumber?: string) {
    this.id = id;
    this.name = name;
    this.address = address;
    this.email = email;
    this.phoneNumber = phoneNumber;
  }

  public id?: number;
  public name?: string;
  public address?: string;
  public email?: string;  
  public phoneNumber?: string;
  public index?: number;
}
